docs/source/en/model_doc/clip.mdx

```diff
@@ -75,6 +75,33 @@ encode the text and prepare the images. The following example shows how to get t
 
 This model was contributed by [valhalla](https://huggingface.co/valhalla). The original code can be found [here](https://github.com/openai/CLIP).
 
+## Resources
+
+A list of official HuggingFace and community (indicated by 🌎) resources to help you get started with CLIP. If you're
+interested in submitting a resource to be included here, please feel free to open a Pull Request and we will review it.
+The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-to-image"/>
+- A blog post on [How to use CLIP to retrieve images from text](https://huggingface.co/blog/fine-tune-clip-rsicd)
+- A blog bost on [How to use CLIP for Japanese text to image generation](https://huggingface.co/blog/japanese-stable-diffusion)
+- A Huggingface space on [Finding best reaction gifs to a given text using CLIP](https://huggingface.co/spaces/flax-community/clip-reply-demo)
```

Comment:
I think we can remove all links to Spaces since these demonstrate an end result instead of showing users how they can get there. Cool idea though!